client/src/Movies/MoviePoster.js

```diff
@@ -0,0 +1,10 @@
+const Poster = [
```

Comment:
Changing this ```Poster``` variable to ```Movieposter``` will fix your problems 👍 You are trying to export something that does not exist on line 10.